node/nodeBlocks.go

```diff
@@ -28,6 +28,10 @@ func (n *Node) GetBlockByNonce(nonce uint64, withTxs bool) (*apiBlock.APIBlock,
 }
 
 func (n *Node) createAPIBlockProcessor() blockAPI.APIBlockHandler {
+	unmarshalTxWrapper := func(txBytes []byte, txType string) (*transaction.ApiTransactionResult, error) {
```

Comment:
Well, nice, but can also be added below, as a regular function :)

Reply:
done.